src/tabs/tabs.js

```diff
@@ -85,7 +85,7 @@ angular.module('ui.bootstrap.tabs', [])
 
   function findTabIndex(index) {
     for (var i = 0; i < ctrl.tabs.length; i++) {
-      if (ctrl.tabs[i].index === index) {
+      if (ctrl.tabs[i].index === +index) {
```

Comment:
This was getting called from line 76 where we were now allowing the value to be a string.

The comparison wouldn't work for 1 === "1".